scripts/ct/run.sh

```diff
@@ -330,8 +330,11 @@ else
                     -e SUITEGROUP="${SUITEGROUP:-}" \
                     -e ENABLE_COVER_COMPILE="${ENABLE_COVER_COMPILE:-}" \
                     -e CT_COVER_EXPORT_PREFIX="${CT_COVER_EXPORT_PREFIX:-}" \
+                    -e NEW_MIX_BUILD=1 \
+                    -e TEST=1 \
+                    -e MIX_ENV="${PROFILE}" \
                     -i $TTY "$ERLANG_CONTAINER" \
-                    bash -c "BUILD_WITHOUT_QUIC=1 make ${WHICH_APP}-ct"
```

Comment:
Couldn't figure out why this was introduced long ago.  :thinking: 
https://github.com/emqx/emqx/commit/f0e03086a639390df54b5698d68ce2a1cbcb9a59